### PR TITLE
fix: Result messages

### LINKED ISSUE #114: Some sucess and error messages not syntax highlighted correctly.

### LINKED ISSUE #118: Success/Error messages should change `persons` to `students`

### LINKED ISSUE #119: More specific error message for `setcrs`
## Currently

The error message format isn't the same as that in the DG.

## Suggestion

See https://github.com/AY2324S2-CS2103T-F13-1/tp/pull/95#discussion_r1541092996

### PR DESCRIPTION
**Pull request by tituschewxj**
### Visible changes
* Change person to student. Resolves #118
* Change address book to contact book.
* Use this format for NUSNet ID: `NUSNet ID`.
* Update `setcrs` message usage. Resolves #119
* `Constraints: ` will appear when there are constraints. This is better as it makes the `Parameters: ` section clearer.
* All messages will be syntax highlighted correctly (SUCCESS/NEUTRAL/ERROR). Resolves #114

### Developer changes
* New `StringUtil` class method to compare stripped strings
* Move command-specific messages to their respective classes
* Update tests
* Command Util Classes
  * Change parameter hint to parameter constraint
  * Add new label constraint
  * Add `NEUTRAL_MESSAGES`

>[!NOTE]
>`edit`
>
>Screenshot 2024-03-31 141631

### CODE REVIEW

File: src/main/java/seedu/address/logic/Messages.java
Comment (saiutkarsh33) [inline src/main/java/seedu/address/logic/Messages.java:1]: Maybe having proper javadoc comments help?

File: src/main/java/seedu/address/logic/commands/UnmarkAttendanceCommand.java
Comment (saiutkarsh33) [inline src/main/java/seedu/address/logic/commands/UnmarkAttendanceCommand.java:1]: More fully fleshed javadocs comments

**Review by saiutkarsh33 — CHANGES_REQUESTED** [review]
Thanks for this PR! my only nit would be to have more fully fleshed out / elaborated javadoc comments.

File: src/main/java/seedu/address/logic/commands/UnmarkAttendanceCommand.java
Comment (tituschewxj) [reply src/main/java/seedu/address/logic/commands/UnmarkAttendanceCommand.java:1]: @saiutkarsh33 I've resolved the issue! Help me review the changes.

**Review by saiutkarsh33 — APPROVED** [review]
LGTM

**Pull request merged**